src/pages/en/guides/deploy/sst.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: Deploy your Astro Site to AWS with SST
+description: How to deploy your Astro site to AWS with SST
+layout: ~/layouts/DeployGuideLayout.astro
+i18nReady: true
+---
+
+You can deploy an Astro site using [SST](https://sst.dev), an open-source framework for deploying fully serverless applications to AWS with SSG and SSR support.
+
+You can also use any additional SST constructs like Cron Jobs, Buckets, Queues, etc while maintaining type-safety.
+
+## Quickstart
+
+1. Create an astro project
+1. Run `npx create-sst@rc`
+1. It should detect that you are using Astro and ask you to confirm.
+1. Once you're ready for deployment you can run `npx sst deploy --stage=production`
+
+To use any [additional SST constructs](https://docs.sst.dev/), add them to `sst.config.ts`. 
+
+```ts {2}
+app.stack(function Site(ctx) {
+  const bucket = new Bucket(ctx.stack, "public");
+  const site = new AstroSite(ctx.stack, "site", {
+    bind: [bucket],
+  });
+
+  ctx.stack.addOutputs({
+    url: site.url,
+  });
+});
+```
+
+Consult the [SST docs on Resource Binding](https://docs.sst.dev/resource-binding) to understand how to use them in your Astro code.
+
+You can also watch [a video walkthrough of this process](https://www.youtube.com/watch?v=AFP3ZHxO7Gg) that will guide you through the steps.
+
+If you have any questions, you can [ask in the SST discord](https://discord.gg/sst).
```

Comment:
```suggestion

You can also watch [a video walkthrough of this process](https://www.youtube.com/watch?v=AFP3ZHxO7Gg) that will guide you through the steps.

### SST constructs
To use any [additional SST constructs](https://docs.sst.dev/), add them to `sst.config.ts`. 

```ts {2}
app.stack(function Site(ctx) {
  const bucket = new Bucket(ctx.stack, "public");
  const site = new AstroSite(ctx.stack, "site", {
    bind: [bucket],
  });

  ctx.stack.addOutputs({
    url: site.url,
  });
});
```

Consult the [SST docs on Resource Binding](https://docs.sst.dev/resource-binding) to understand how to use them in your Astro code.

If you have any questions, you can [ask in the SST Discord](https://discord.gg/sst).
```
Ah! Had a last minute change of heart checking the preview! 

- capitalize Discord
- Walkthrough after the steps that it actually walks through?
- A header to separate SST contructs visually

Any concerns here?